### DISCUSSION TITLE
Phone number authorization

### DISCUSSION THREAD

**Discussion by catative** [wp-graphql/wp-graphql#3265] [General]
Is it possible to implement authorization by phone number in headless mode similar to email? Are there any ready-made solutions on this topic? Or can you please suggest a direction for a solution (any thoughts)? Currently I am using JWT authorization via a plugin: https://github.com/wp-graphql/wp-graphql-jwt-authentication

**Comment by justlevine** [comment]
https://github.com/AxeWP/wp-graphql-headless login allows you to use PHP to register custom authentication providers.

You'd need to handle mapping the phone numbers to WP users though (this is not natively supported by WordPress).
